src/org/thoughtcrime/securesms/ApplicationContext.java

```diff
@@ -77,7 +79,15 @@ public void onCreate() {
 
     dcAccounts = new DcAccounts("Android "+BuildConfig.VERSION_NAME, new File(getFilesDir(), "accounts").getAbsolutePath());
     AccountManager.getInstance().migrateToDcAccounts(this);
-    if (dcAccounts.getAll().length == 0) {
+    int[] allAccounts = dcAccounts.getAll();
+    for (int accountId : allAccounts) {
```

Comment:
@link2xt @Hocuri I wonder if this is something we want to let the account manager handle, by passing the passphrase to the account manager